Sources/FuzzilliCli/TerminalUI.swift

```diff
@@ -51,7 +51,7 @@ class TerminalUI {
         fuzzer.registerEventListener(for: fuzzer.events.CrashFound) { crash in
             if crash.isUnique {
                 print("########## Unique Crash Found ##########")
-                print(fuzzer.lifter.lift(crash.program))
+                print(fuzzer.lifter.lift(crash.program, withOptions: []))
```

Comment:
Why is the `withOptions:` added here? It should just call https://github.com/googleprojectzero/fuzzilli/blob/0e4935efd3ba2d860c2d1647be25f5a333909719/Sources/Fuzzilli/Lifting/Lifter.swift#L21 no?